some.json

```diff
@@ -0,0 +1,13 @@
+{
+  "email": "",
+  "prBoard": "",
+  "comment": {
+    "title": "",
+    "description": "",
+    "board": "",
```

Comment:
/ticket change field name -d should be bored

Reply:
Here's your ticket: [ change field name](https://ticketgen19.atlassian.net/browse/IKKI-49)